setup.py

```diff
@@ -126,7 +126,8 @@ def get_exts_for(name):
         install_requires=[
             "enum34; python_version<'3.4'",
             "funcsigs>=1.0.0; python_version=='2.7'",
-            "msgpack>=0.5.0",
+            "msgpack>=0.5.0,<1.0; python_version<'3.0'",
```

Comment:
I had to force this as it turns out that the tracer when run with Python 2 under `tox` with `msgpack==1.0` causes some bizarre encoding behaviour. This strange encoding actually makes the traces undecodable by both the test agent and the real agent.

Reply:
Add a test in a different PR with this?

Reply:
weird... should we try to get #1491 merged in, since that will make this issue moot?

that draft pr should mostly be good to go, no?

Reply:
Yeah, made the change just here to be able to test the CI properly. I'll look into whether this is reproducible or not and open a subsequent PR. Also #1491 will solve this.